internal/api/installation.go

```diff
@@ -661,7 +669,15 @@ func handleDeleteInstallation(c *Context, w http.ResponseWriter, r *http.Request
 		return
 	}
 
-	err = c.EventProducer.ProduceInstallationStateChangeEvent(installationDTO.Installation, oldState)
+	// Try to parse the user ID from the request context, so it can be passed along with the webhook event
+	actorID := ""
+	if value := r.Context().Value(ContextKeyUserID{}); value != nil {
+		if str, ok := value.(string); ok && str != "" {
+			actorID = str
+		}
+	}
+
+	err = c.EventProducer.ProduceInstallationStateChangeEvent(installationDTO.Installation, oldState, events.DataField{Key: "actor_id", Value: actorID})
```

Comment:
I snuck this in while I was here. This passes the ID of the actor that initiated the deletion request, so that webhook listeners can easily detect who did the deletion. The main use case for this will be the Cloud Plugin. Currently, whenever an installation is deleted for any reason, a message "Cloud isntallation <> has been manually deleted and is now removed from the cloud plugin", which confuses users, as sometimes an automatic cleanup script deletes them. With this change, the Cloud Plugin will be able to tell if it itself made the deletion request, or if it was another system, and adjust the message sent accordingly.